docs/lib/push-notifications/fragments/js/getting-started.md

```diff
@@ -174,17 +174,10 @@ Push Notifications category is integrated with [AWS Amplify Analytics category](
     $ react-native init myapp
     $ cd myapp
     $ npm install
-    $ npm install aws-amplify && npm install @aws-amplify/pushnotification
-    $ react-native link @aws-amplify/pushnotification
-    $ react-native link amazon-cognito-identity-js # link it if you need to Sign in into Cognito user pool
+    $ npm install aws-amplify && npm install @aws-amplify/pushnotification && npm install @react-native-community/push-notification-ios
```

Comment:
I think line wrapping would help comprehension:

```bash
 npm install aws-amplify && \
    npm install @aws-amplify/pushnotification && \
    npm install @react-native-community/push-notification-ios
```

Or just make them separate lines because, why not?

Reply:
chose line seperation and updated